### PR TITLE
Fix sass compilation

### LINKED ISSUE #2523: scss compiles error
image
recurrent trouble
git clone,and change files 
Counter.css->Counter.scss
Counter.tsx ->
```js
import React from 'react';
import { useSelector, useDispatch } from 'react-redux';
// import { Link } from 'react-router-dom';
import './Counter.scss';
// import routes from '../../constants/routes.json';
import {
  // increment,
  // decrement,
  // incrementIfOdd,
  // incrementAsync,
  selectCount,
} from './counterSlice';

export default function Counter() {
  // const dispatch = useDispatch();
  // const value = useSelector(selectCount);
  return (
    <div>

    </div>
  );
}
```
yarn dev ,and error
- [ 1.19.1 ] Using yarn
- [ 12.14.1 ] Using node 10.x

"name": "css-loader",
"version": "3.6.0",
image

### PR DESCRIPTION
**Pull request by jooohhn**
Fixes #2523

### CODE REVIEW

File: configs/webpack.config.renderer.dev.babel.js [hunk 1]
```diff
@@ -118,10 +118,13 @@ export default merge(baseConfig, {
         test: /^((?!\.global).)*\.(scss|sass)$/,
         use: [
           {
-            loader: 'typings-for-css-modules-loader',
+            loader: 'style-loader',
+          },
+          {
+            loader: '@teamsupercell/typings-for-css-modules-loader',
```
Comment (amilajack) [inline configs/webpack.config.renderer.dev.babel.js:124]: I don't know the context of the fix. Can you provide some context on how this fixes the bug

Comment (jooohhn) [reply configs/webpack.config.renderer.dev.babel.js:124]: I think there are 2 parts to this fix
- Re-added node-sass which was accidentally removed in v1.2.0
- Replaced `typings-for-css-modules-loader` with an actively maintained fork
  - [See thread](https://github.com/Jimdo/typings-for-css-modules-loader/issues/94)

**Comment by jooohhn** [comment]
> Do you know what's causing this? https://github.com/electron-react-boilerplate/electron-react-boilerplate/pull/2540/checks?check_run_id=935900755#step:5:19

@amilajack not sure what's up with the `package-ci\nyarn` error. It looks like it's been in CI since [this commit](https://github.com/electron-react-boilerplate/electron-react-boilerplate/runs/903967552#step:5:19)

File: configs/webpack.config.renderer.dev.babel.js (hunk 1 above)
Comment (amilajack) [reply configs/webpack.config.renderer.dev.babel.js:124]: I see, makes sense. Good call!

**Pull request merged**